pkgs/aws/aws-cli/registry.yaml

```diff
@@ -0,0 +1,17 @@
+packages:
+  - type: http
+    repo_owner: aws
+    repo_name: aws-cli
+    version_source: github_tag
+    url: "https://awscli.amazonaws.com/awscli-exe-{{.OS}}-{{.Arch}}-{{.Version}}.zip"
```

Comment:
```suggestion
    url: https://awscli.amazonaws.com/awscli-exe-{{.OS}}-{{.Arch}}-{{.Version}}.zip
```

Reply:
https://aquaproj.github.io/docs/reference/registry-style-guide#remove-unneeded-quotes-of-strings